sxr-arpet/app/build.gradle

```diff
@@ -42,7 +42,7 @@ android {
         buildConfigField "String", "GOOGLE_CLOUD_ANCHOR_KEY_NAME", "\"$GOOGLE_CLOUD_ANCHOR_KEY_NAME\""
         manifestPlaceholders = [cloudAnchorApiKeyName : GOOGLE_CLOUD_ANCHOR_KEY_NAME,
                                 cloudAnchorApiKeyValue: getPropertyOrDefault("cloudAnchorApiKey", "")]
-        minSdkVersion 26
+        minSdkVersion 21
```

Comment:
Why 21 and not 26?

Reply:
@liaxim It did not work on my end until I changed the version. Now I have set it back.